common/problemsettings.go

```diff
@@ -269,8 +269,9 @@ type InputsValidatorSettings struct {
 // TestsSettings represent the tests that are to be run against the problem
 // itself. They are stored in tests/settings.json.
 type TestsSettings struct {
-	Solutions       []SolutionSettings       `json:"solutions"`
-	InputsValidator *InputsValidatorSettings `json:"inputs,omitempty"`
+	Solutions        []SolutionSettings       `json:"solutions"`
+	InputsValidator  *InputsValidatorSettings `json:"inputs,omitempty"`
+	ExpectedMaxScore int64                    `json:"maxScore,omitempty"`
```

Comment:
esto debería ser un racional por consistencia con cómo se expresan los puntajes en `SolutionSettings.ScoreRange`.

```suggestion
	ExpectedMaxScore *big.Rat                 `json:"maxScore,omitempty"`
```

[`big.Rat`](https://pkg.go.dev/math/big#Rat) implementa [`MarshalText`](https://pkg.go.dev/math/big#Rat.MarshalText), así que si se cadenifica el puntaje máximo, debería funcionar como entero _o_ como fracción.

Reply:
pensándolo bien, mejor usa este: https://github.com/omegaup/go-base/pull/17 (`*base.Rat`). así los casos de prueba se pueden quedar igual.

Reply:
oh claro, hay que correr `go get github.com/omegaup/go-base/v3@v3.2.4` para poder usar esto.